governance.md

```diff
@@ -57,6 +57,7 @@ Council is to ensure, through taking input from the Community, the long-term
 well-being of The Project, both technically and as a community. As The Project
 draws its strength from representing the Community, it strives to avoid
 fragmentation of the Community into separate projects, if possible.
+The Steering Council will maintain a minimum of three members.
```

Comment:
Required by NumFocus agreement.

Reply:
It's not critical, but if it's not too late, seem like something like 7 members should be the minimum for the Steering Council (3 seems too small to me).

Reply:
I suggest we revisit this later. In practice, we have more than 7 today so this shouldn't be a blocker to getting started with NumFocus.

Reply:
Okay